pci/src/configuration.rs

```diff
@@ -291,7 +291,7 @@ pub enum PciExpressCapabilityId {
     VfResizeableBar = 0x0024,
     DataLinkFeature = 0x0025,
     PhysicalLayerSixteenGts = 0x0026,
-    LaneMargeningAtTheReceiver = 0x0027,
+    LaneMarginingAtTheReceiver = 0x0027,
```

Comment:
This is probably an api break, but I think it's correct

Reply:
It's fine.